resharper/resharper-unity/src/Yaml/Psi/DeferredCaches/AssetInspectorValues/Values/AssetReferenceValue.cs

```diff
@@ -56,13 +59,27 @@ public override int GetHashCode()
         public string GetPresentation(ISolution solution, IDeclaredElement declaredElement, bool prefabImport)
         {
             solution.GetComponent<IShellLocks>().AssertReadAccessAllowed();
+            var hierarchyContainer = solution.GetComponent<AssetDocumentHierarchyElementContainer>();
+
+            if (UnityApi.IsDescendantOfScriptableObject((declaredElement as IField)?.Type.GetTypeElement()))
+            {
+                if (Reference is LocalReference localReference && localReference.LocalDocumentAnchor == 0)
+                    return "None";
+                
+                var sourceFile = hierarchyContainer.GetSourceFile(Reference, out _);
+                if (sourceFile == null)
+                    return "...";
 
+                var relativePath = sourceFile.DisplayName.Replace('\\', '/').RemoveStart("Assets/");
```

Comment:
What happens with assets in packages?

Reply:
I am not sure, that we provide support for assets in packages, will check and fix it. For packages, we will show path, which starts from "packages/"